MohammadBallal/task_manager.py

```diff
@@ -0,0 +1,120 @@
+from task import Task, UrgentTask
+from utils import validate_date, get_valid_input, valid_priority ,validate_status
+
+
+regularTasks= []
+urgentTasks= []
+
+def add_task():
+    #using get_valid_input for all user inputs to validate it
+    title= get_valid_input("What is the task title? : ")
+    description= get_valid_input("Just a little description for the task here please : ")
+    due_date= get_valid_input("Enter task due date (DD/MM/YYYY)")
+    #validate date format
+    validate_date(due_date)
+    #check if the task is urgent or not to know where to store it 
+    isUrgent= get_valid_input("Is this task urgent (please write : yes || no) :  ").lower() #To reduce user input errors
+    if isUrgent == "yes" :
+        priority= get_valid_input("Enter task priority ( High || Medium || Low ) : ").lower()
+        #validate priority
+        valid_priority(priority)
+        task= UrgentTask(title,description,due_date,priority)
+        urgentTasks.append(task)
+    else :
+        task= Task(title,description,due_date)
+        regularTasks.append(task)
+    
+    print("TASK ADDED SUCCESSFULLY")
+
+
+def view_tasks():
+    #Check if there any reguler tasks stored 
+    if regularTasks:
```

Comment:
do we care if it's regular or urgent, why don't having a single list, the two classes implanted the display method, and if we want to check if it's an instance of a class you can use `isinstance` function

Reply:
I simply separated the cases so that the results would be displayed in more detail. For example if there are only urgent tasks it will show them and mention that there are no regular tasks, and so on.